creation/web_base/condor_startup.sh

```diff
@@ -641,9 +649,9 @@ LSB_DISTRIBUTOR_ID="UNKNOWN"
 LSB_DESCRIPTION="UNKNOWN"
 command -v lsb_release >/dev/null
 if test $? = 0; then
-    LSB_RELEASE=`lsb_release -rs | sed 's/"//g'`
-    LSB_DISTRIBUTOR_ID=`lsb_release -is | sed 's/"//g'`
-    LSB_DESCRIPTION=`lsb_release -ds | sed 's/"//g'`
+    LSB_RELEASE=$(lsb_release -rs | sed 's/"//g')
+    LSB_DISTRIBUTOR_ID=$(lsb_release -is | sed 's/"//g')
+    LSB_DESCRIPTION=$(lsb_release -ds | sed 's/"//g')`
```

Comment:
The closing backtick needs to be removed (was retained after the closing parantheses)